pkg/manifests/external_dns.go

```diff
@@ -184,13 +204,12 @@ func newExternalDNSDeployment(conf *config.Config, configMapHash string, externa
 					Containers: []corev1.Container{*withLivenessProbeMatchingReadiness(withTypicalReadinessProbe(7979, &corev1.Container{
 						Name:  "controller",
 						Image: path.Join(conf.Registry, "/oss/kubernetes/external-dns:v0.11.0.2"),
-						Args: []string{
-							"--provider=" + provider,
+						Args: append([]string{
+							"--provider=" + externalDnsConfig.Provider,
 							"--source=ingress",
 							"--interval=3m0s",
-							"--txt-owner-id=" + externalDnsConfig.RecordId,
-							"--domain-filter=" + externalDnsConfig.Domain,
-						},
+							"--txt-owner-id=" + operatorName,
```

Comment:
using the kube-system uid as a unique identifier per [this](https://github.com/kubernetes/kubernetes/issues/77487#issuecomment-489786023) comment I saw in a thread about this issue.  

My one concern is needing a fallback issue in case getting the ns fails but not really sure what to do about that